pkg/cloudprovider/providers/photon/photon.go

```diff
@@ -0,0 +1,550 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package photon
```

Comment:
I think it's worth having a comment here about what cloud provider interfaces we're providing. This is only the disk (volume?) interface, not load balancer, routes, etc.

Also, this can be configured through a file--having a note about the expected location of that file is probably good.